docs/src/samples/monitoring/alerts.yaml

```diff
@@ -55,3 +55,12 @@ groups:
     for: 1m
     labels:
       severity: warning
+  - alert: ReplicaFailingReplication
+    annotations:
+      description: Replica {{ $labels.pod }} is failing to replicate
+      summary: Checks if the replica is failing to replicate
+    expr: |-
+      (1 - cnpg_pg_replication_in_recovery + cnpg_pg_replication_is_wal_receiver_up) < 1
```

Comment:
Not familiar with Prometheus alerts, but why not:

```
cnpg_pg_replication_in_recovery > cnpg_pg_replication_is_wal_receiver_up
```

Reply:
Yup, works will change accordingly 👍 .